### ISSUE TITLE
Setting negative audio delay using text field is not working

### ISSUE THREAD

**Issue by saneef** [iina/iina#4482]
**System and IINA version:**

- macOS 13.4.1
- IINA 1.3.2 Build 135

**Expected behavior:**
Setting a delay of negative value, like `-1`, using text field should apply the value.

**Actual behavior:**
Negative values are ignored.

<br>

**Steps to reproduce:**

* Open a video.
* Open Audio panel, using the menu, 'Audio > Show Audio Panel'.
* Type in a delay of negative value, like `-2`

**How often does this happen?**

_Everytime._

**Comment by low-batt** [comment]
Thank you very much for reporting this.

Reproduced for me. I confirmed this is working in IINA 1.3.1. This is a regression.

A brief look at the quick settings view controller XIB shows the NSNumberFormatter for the audio delay is configured with a `0.01` minimum value whereas the similar formatter for subtitle delay, which is working, has no minimum configured. I believe the setting of the minimum restriction was added in commit 64d488311f743de682db21a6b157e7608bfc1801.
